### ISSUE TITLE
XIAOMI Outdoor AW300

### ISSUE THREAD

**Issue by dantasjoao** [TheCrypt0/yi-hack-v4#376]
Hello
Can you tell me if it works on Xiaomi AW300?

Thanks and great project 😄

**Comment by rzfzr** [comment]
Yeah, I was also curious if by Yi 1080p Outdoor it meant the AW200, the 300, or something else
